VAMobile/documentation/design/Foundation/Design tokens/Color.md

```diff
@@ -10,10 +10,52 @@ Our color palette is organized into semantic color tokens, which have specific m
 To learn more about color tokens, see the [slides](https://docs.google.com/presentation/d/1grihSnsCPHBidOhe2T-sAC2HUo1_aU54XN8mqmyYySw/edit#slide=id.g26f52ea1451_0_0) from Sprint Demo on May 7, 2024.
 
 ## Primitive
+**Open in**: [Storybook](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/docs/design-tokens-colors--docs#primitive)
 <iframe width="800" height="1200" alt="Image of design tokens in Figma" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com/design/rdLIEaC9rVwX70QbIGkMvG/%F0%9F%93%90-Design-Tokens-Library---Design-System---VA-Mobile?node-id=570-942&t=Ko6aa9sM96UxylvN-4" allowfullscreen></iframe>
 
 ## Semantic
-<iframe width="800" height="1200" alt="Image of design tokens in Figma" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com/design/rdLIEaC9rVwX70QbIGkMvG/%F0%9F%93%90-Design-Tokens-Library---Design-System---VA-Mobile?node-id=570-3391&t=Ko6aa9sM96UxylvN-4" allowfullscreen></iframe>
+**Open in**: [Storybook](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/docs/design-tokens-colors--docs#semantic)
+<iframe width="800" height="3000" alt="Image of design tokens in Figma" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com/design/rdLIEaC9rVwX70QbIGkMvG/%F0%9F%93%90-Design-Tokens-Library---Design-System---VA-Mobile?node-id=1239-695&t=PljikYyjG5LVlwDo-4" allowfullscreen></iframe>
```

Comment:
I'm assuming the `node-id` change might be to fix it, but just noting that [looking at the live site I noticed the Semantic section](https://department-of-veterans-affairs.github.io/va-mobile-app/design/Foundation/Design%20tokens/Color#semantic) is currently pointing to the Figma cover page in the iframe.